code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -20,35 +20,40 @@
 	skinned_type = /obj/item/stack/sheet/animalhide/human
 	var/info_text = "You are a <span class='danger'>Vampire</span>. You will slowly but constantly lose blood if outside of a coffin. If inside a coffin, you will slowly heal. You may gain more blood by grabbing a live victim and using your drain ability."
 	var/obj/effect/proc_holder/spell/targeted/shapeshift/bat/batform //attached to the datum itself to avoid cloning memes, and other duplicates
+	var/halloween_version = FALSE //SKYRAT EDIT: Allow a neutered version of vampires without batform
 
 /datum/species/vampire/check_roundstart_eligible()
-	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
-		return TRUE
-	return FALSE
+	//if(SSevents.holidays && SSevents.holidays[HALLOWEEN]) SKYRAT EDIT - ALL YEAR :smiling_imp:
+	return TRUE
+	//return FALSE SKYRAT EDIT - ALL YEAR :smiling_imp:
 
 /datum/species/vampire/on_species_gain(mob/living/carbon/human/C, datum/species/old_species)
 	. = ..()
 	to_chat(C, "[info_text]")
-	C.skin_tone = "albino"
+	// C.skin_tone = "albino" SKYRAT EDIT: Allow vampires to be different skin tones beside one
 	C.update_body(0)
-	if(isnull(batform))
+	//SKYRAT EDIT: Allow a neutered version of vampires without batform
+	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
+		halloween_version = TRUE
+	if(isnull(batform) && halloween_version)
+	//SKYRAT EDIT: Allow a neutered version of vampires without batform
 		batform = new
 		C.AddSpell(batform)
 	C.set_safe_hunger_level()
 
 /datum/species/vampire/on_species_loss(mob/living/carbon/C)
 	. = ..()
-	if(!isnull(batform))
+	if(!isnull(batform) && halloween_version) //SKYRAT EDIT: Allow a neutered version of vampires without batform
```

Comment:
I don't think the halloween_version check is needed here. It's going to be null if the halloween_version is FALSE anyway.